robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+
+*** Test Cases ***
+
+User On A Non RD2 Enbaled Org should Not See Status Mappings Option
+    [Documentation]               On an org without rd2 enabled, a user
+     ...                          Should not see the option Status to state mapping option
+     ...                          Under NPSP Settings
+
+
+    [tags]                       unstable                      W-8211276
+
+    Go To Page                   Custom                        NPSP_Settings
+    Open Main Menu               Recurring Donations
+    Verify Sub Link Present      Status to State Mapping       False
```

Comment:
this test case looks very short, cant this be combined with any other test case?

Reply:
I know, But that is how we decided to create and pointed the workitem accordingly. This is the beginning for a series of field mapping automation stories to follow.

Reply:
so will u be adding more to this test case later on in future sprints?

Reply:
This test case is a valid scenario and is as is part of the validation as we do not want this sub menu option to be present when user has no RD2 enabled. The following ones are for scenarios where rd2 is enabled and user sees the submenu options and the following actions.